src/mpi/session/session_impl.c

```diff
@@ -5,16 +5,12 @@
 
 #include "mpir_session.h"
 #include "mpi_init.h"
+#include "mpir_pset.h"
```

Comment:
Split the changes for `session_impl` into a separate commit as well. The changes in `group_impl.c` are a key internal mechanism. The changes here are more a surface-layer implementations :)

Reply:
Done.